docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -8,36 +8,48 @@ tags: ["data center", "nutanix"]
 ---
 
 
-[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a technical preview implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette. Integration with Nutanix is achieved through a Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix.
+[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a Tech Preview of implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette. Integration with Nutanix is achieved through a Self-hosted Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix Prism Central and allows Palette to monitor Nutanix clusters.
 
-The section below describes the overall workflow to register Nutanix with Palette and deploy a cluster using Palette. 
+## Get Started
+
+Learn how to deploy a cluster to Nutanix by using Palette. Check out the [Deploy a Cluster with Palette](../../public-cloud/deploy-k8s-cluster.md) tutorial to get started.
+
+The section below describes the workflow to register Nutanix with Palette and deploy a cluster. 
 
 ## Workflow
 
-The following summary steps outline the CAPI cloud workflow illustrated in the diagram below. A system administrator performs the first step at the system level to make the cloud provider available to all the tenants. CAPI users perform the remaining steps in Palette and their local environment.
+The following summary steps outline the Nutanix cloud workflow illustrated in the diagram. Since only self-hosted instances of Palette and Palette VerteX have access to the system console by a system administrator, the registration step is performed at the system level, making the Nutanix cloud available to all the tenants. Regular Palette users perform the remaining steps in Palette and their local environment.
```

Comment:
```suggestion
The following summary steps outline the Nutanix cloud workflow illustrated in the diagram. Since only self-hosted instances of Palette and Palette VerteX have access to the system console by a system administrator, the registration step is performed at the system level, making the Nutanix cloud available to all the tenants. Regular Palette or VerteX users perform the remaining steps in Palette and their local environment.
```

Reply:
:white_check_mark: